Dockerfile

```diff
@@ -0,0 +1,35 @@
+# A container that launches the Polymesh node in unsafe development mode with an open WS port 9944.
+#
+# Build with `docker build . -t polymesh`
+
+FROM rust:latest
+
+RUN apt-get update && apt-get upgrade -y && \
```

Comment:
this can be:
```
RUN apt-get update && \
    apt-get install -y \
        gcc \
        g++ \
        pkg-config \
        cmake \
        libssl-dev \
        git \
        clang \
        libclang-dev
```
`aptitude` brings a lot along with it